docs: describe solution in FAQ for slow datetime arrays (#184)
predict-idlab/plotly-resampler: docs/sphinx/FAQ.rst

.. image:: _static/datashader.png

.. raw:: html

   </div>
   </details>
   <br>
   <details>
   <summary>
      <a><b>Pandas or numpy datetime works much slower than unix epoch timestamps?</b></a>
   </summary>
   <div style="margin-left:1em">

> This stems from the plotly scatter(gl) constructor being much slower for non-numeric data. Plotly performs a different serialization for datetime arrays (which are interpreted as object arrays).
> However, plotly-resampler should not be limited by this - to avoid this issue, add your datetime data as `hf_x` to your plotly-resampler ``FigureResampler.add_trace`` (or ``FigureWidgetResampler.add_trace``) method.
> This avoids adding (& serializing) *all* the data to the scatter object. Now plotly-resampler will only add the necessary resampled data to the scatter object.

Reviewer: ```suggestion
This stems from the plotly scatter(gl) constructor being much slower for non-numeric data. Plotly performs a different serialization for datetime arrays (which are interpreted as object arrays).
However, plotly-resampler should not be limited by this - to avoid this issue, add your datetime data as `hf_x` to your plotly-resampler ``FigureResampler.add_trace`` (or ``FigureWidgetResampler.add_trace``) method.
This avoids adding (& serializing) *all* the data to the scatter object, since  plotly-resampler will pass the aggregated data to the scatter object.
```
